sources/mysql/data_test.go

```diff
@@ -320,6 +320,8 @@ func buildConv(spTable ddl.CreateTable, srcTable schema.Table) *internal.Conv {
 	conv := internal.MakeConv()
 	conv.SpSchema[spTable.Name] = spTable
 	conv.SrcSchema[srcTable.Name] = srcTable
+	conv.ToSource[spTable.Name] = internal.NameAndCols{Name: srcTable.Name, Cols: make(map[string]string)}
+	conv.ToSpanner[srcTable.Name] = internal.NameAndCols{Name: spTable.Name, Cols: make(map[string]string)}
```

Comment:
Why did we have to modify this?

Reply:
This change was already there but was removed as part of https://github.com/cloudspannerecosystem/harbourbridge/pull/204/files .